### PR TITLE
Add MsBuildResolver and CMakeResolver examples

### PR DESCRIPTION
**Pull request by jbayardo**

### CODE REVIEW

File: Examples/MsBuildHelloWorld/config.bc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+config({
+    resolvers: [
+        {
+            kind: "MsBuild",
+            root: d`.`,
```
Comment (dannyvv) [inline Examples/MsBuildHelloWorld/config.bc]: side-note: Should we rename this to projectRoot like cmake?

Comment (smera) [reply Examples/MsBuildHelloWorld/config.bc]: We could... don't have a strong opinion about this :)

---
In reply to: [285703454](https://github.com/microsoft/BuildXL/pull/340#discussion_r285703454) [](ancestors = 285703454)

Comment (cdmihai) [reply Examples/MsBuildHelloWorld/config.bc]: Inconsistency raises the WTF/minute metric :)

---
In reply to: [285705018](https://github.com/microsoft/BuildXL/pull/340#discussion_r285705018) [](ancestors = 285705018,285703454)

**Comment by dannyvv** [comment]
Not needed in this PR, but we should file a workitem to have these examples build as part of our unittests.

**Pull request merged**